modules/fedimint-mint-client/src/lib.rs

```diff
@@ -670,6 +674,7 @@ impl ClientModule for MintClientModule {
 
     fn context(&self) -> Self::ModuleStateMachineContext {
         MintClientContext {
+            client_ctx: self.client_ctx.clone(),
```

Comment:
The state machine system also has a global context object, would that be the better place to put this?

Reply:
Oh, makes sense! Didn't consider client context only being available for modules.